src/app/pages/tasks/tasks.component.ts

```diff
@@ -3,25 +3,37 @@ import { Component, OnInit } from '@angular/core';
 @Component({
   selector: 'app-tasks',
   templateUrl: './tasks.component.html',
-  styleUrls: ['./tasks.component.css']
+  styleUrls: ['./tasks.component.scss']
 })
 export class TasksComponent implements OnInit {
+  taskName: { id:number, name:string, completed:boolean }[] = [];
+  checkBoxColor = 'primary';
+  completeList:{id:number, name:string, completed:boolean}[] = [];
```

Comment:
add space
1. after colon
2. and after curly brace open
3. before curly brace close